go/test/endtoend/cluster/cluster_process.go

```diff
@@ -667,6 +674,18 @@ func (cluster *LocalProcessCluster) NewVttabletInstance(tabletType string, UID i
 	}
 }
 
+// NewOrcProcess creates a new OrchestratorProcess object
+func (cluster *LocalProcessCluster) NewOrcProcess(configFile string) *OrchestratorProcess {
+	base := VtctlProcessInstance(cluster.TopoProcess.Port, cluster.Hostname)
+	base.Binary = "orchestrator"
+	return &OrchestratorProcess{
+		VtctlProcess: *base,
+		LogDir:       cluster.TmpDirectory,
+		//ExtraArgs: cluster.OrcExtraArgs,
```

Comment:
All extraneous fields will be cleaned up before I move this PR from Draft. I kept that around in case I need it for one of the cases that hasn't been implemented yet.